client/src/main/java/org/opensearch/ml/client/MachineLearningNodeClient.java

```diff
@@ -154,6 +160,14 @@ public void searchModel(SearchRequest searchRequest, ActionListener<SearchRespon
         }, listener::onFailure));
     }
 
+    @Override
+    public void searchModelGroup(SearchRequest searchRequest, ActionListener<SearchResponse> listener) {
```

Comment:
Why do we need to expose this API in ML client?

Reply:
Sorry my bad. Need to remove it from here. I was making changes similar to that of search model initially and forgot to delete it from here